tests/inspectdb/models.py

```diff
@@ -106,6 +106,13 @@ class Meta:
         required_db_features = {"supports_collation_on_textfield"}
 
 
+supports_unlimited_varchar = connection.features.supports_unlimited_varchar
+
+
+class CharFieldUnlimitedVarchar(models.Model):
+    char_field = models.CharField(max_length=None if supports_unlimited_varchar else 1)
```

Comment:
We should use  `Meta.required_db_features` instead:
```suggestion
class CharFieldUnlimitedVarchar(models.Model):
    char_field = models.CharField(max_length=None)

    class Meta:
        required_db_features = {"supports_unlimited_varchar"}
```